solutions/usaco-966.mdx

```diff
@@ -47,4 +47,46 @@ ans += first8[N % 8]
 print(ans)
 ```
 </PySection>
+<JavaSection>
+  
+```java
+
+import java.util.*;
+import java.lang.*;
```

Comment:
```suggestion

```
do we really need it?

Reply:
No, java. lang package is a default package in Java. Hence, we don't really need to import it explicitly.